app/models/user.rb

```diff
@@ -156,6 +157,10 @@ def self.skip_mx_check?
     Rails.env.local?
   end
 
+  def set_role_id
+    self.role_id ||= -99
+  end
+
   def role
     if role_id.nil?
       UserRole.everyone
```

Comment:
I believe that `UserRole.everyone` can be refactored / reworked, considering that there's no need to lazily create this role in the database.